react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+export default class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      play: false,
+      pause: true,
```

Comment:
These are always opposites, try combining them into `isStarted` (a boolean) or something like that.